schemas/core/reask.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$id": "https://raw.githubusercontent.com/guardrails-ai/interfaces/core-schemas/schemas/core/reask.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "ReAsk",
+  "type": "object",
+  "properties": {
+    "incorrectValue": {
```

Comment:
where do we keep the reaskPrompt? I don't see it in fail-result or validation-result. Is it an additionalPropery?

Reply:
It's currently stored on the schema, once these changes are implemented they go into an execution options container on the Guard.  Will probably become runtime/execution-time only options soon